File: proj/code-repair/mycode/clean_dataset/patch-single-line/quixbugs-java/codex-sh/sqrt.java
--- 
+++ 
@@ -1,7 +1,7 @@
 public static double sqrt(double x, double epsilon) {
-    double approx = x / 2f;
-    while (Math.abs(x-approx) > epsilon) {
-        approx = 0.5f * (approx + x / approx);
+    double approx = x / 2;
+    while (Math.abs(approx * approx - x) > epsilon) {
+        approx = (approx + x / approx) / 2;
     }
     return approx;
 }

File: proj/code-repair/mycode/clean_dataset/patch-single-line/quixbugs-python/incoder-6/sqrt.py
--- 
+++ 
@@ -1,5 +1,5 @@
 def sqrt(x, epsilon):
     approx = x / 2
-    while abs(x - approx) > epsilon:
+    while abs(approx ** 2 - x) > epsilon:
         approx = 0.5 * (approx + x / approx)
     return approx
